.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 
 python:
-  - "3.4"
-  - "3.5"
-  - "3.6"
```

Comment:
This version 3.6 should stay compatible

Reply:
Only if you assume to support versions which have reached EOL: https://endoflife.date/python
Additionnally, as spotted by the linter ([see here](https://github.com/jiri-otoupal/pycrosskit/runs/6373407958?check_suite_focus=true#step:5:13), some syntax is only available since Python 3.8 (walrus op): https://realpython.com/python38-new-features/#the-walrus-in-the-room-assignment-expressions